torch/_inductor/tiling_utils.py

```diff
@@ -320,6 +327,9 @@ def get_node_splits(self) -> tuple[Split, Split]:
         if len(self.all_node_sizes) == 1:
             return next(iter(self.all_node_sizes))
 
+        if len(self.pw_split_options) == 0:
+            return ((self.pointwise_numel,), (self.red_numel,))
```

Comment:
Similarly, this might have happened if we did a poor job finding splits, because of potential sympy difficulty in inferring divisibility